.env.example

```diff
@@ -1,17 +1,25 @@
-# bancho.py supports both INET and UNIX sockets
-# INET: provide an IP for SERVER_ADDR, and a port for SERVER_PORT
-# UNIX: provide a filesystem path for SERVER_ADDR, and no SERVER_PORT
-SERVER_ADDR=/tmp/bancho.sock
-SERVER_PORT=
+# for nginx reverse proxy to work through the containers, the bancho
+# server must expose itself on port 80 to be accessed on http://bancho.
+SERVER_ADDR=0.0.0.0
```

Comment:
unsure if i would go about putting default addr & port to docker values as not everyone will be using it. i’d atleast keep the comments explaining how to use unix/inet there too

Reply:
i think we can probably expect everyone to use this going forward - was planning on updating the readme as well